metaserver/src/main/scala/scala/meta/languageserver/Effects.scala

```diff
@@ -15,4 +15,6 @@ object Effects {
   final val IndexSourcesClasspath = new IndexSourcesClasspath
   final class InstallPresentationCompiler extends Effects
   final val InstallPresentationCompiler = new InstallPresentationCompiler
+  final class PublishLinterDiagnostics extends Effects
+  final val PublishLinterDiagnostics = new PublishLinterDiagnostics
```

Comment:
Should these be objects? Unit like marker effects feel like singletons to me.

Reply:
They're vals to avoid `PublishLinterDiagnostics.type`

Reply:
But you can consider them as singleton objects